ingest/src/main/scala/hydra.ingest/modules/Routes.scala

```diff
@@ -22,7 +22,8 @@ final class Routes[F[_]: Sync: Futurable] private(programs: Programs[F], algebra
     val topicDetails =
       TopicDetails(
         cfg.createTopicConfig.defaultNumPartions,
-        cfg.createTopicConfig.defaultReplicationFactor
+        cfg.createTopicConfig.defaultReplicationFactor,
+        Map("cleanup.policy" -> "compact")
```

Comment:
This would make all topics compacted instead of just the metadata one. The topic details for the metadata topic can be found in `Bootstrap.scala`